core/os/android/adb/commands.go

```diff
@@ -99,6 +99,36 @@ func (b *binding) InstallAPK(ctx context.Context, path string, reinstall bool, g
 	return b.Command("install", args...).Run(ctx)
 }
 
+// PushOBB places a OBB file to the correct location for an APK to access.
+func (b *binding) PushOBB(ctx context.Context, package_name string, version_code int32, obb_path string) error {
```

Comment:
Should this be a method of `adb.InstalledPackage` ?

Don't use underscores in go variable names. Please use golint and govet as part of your development environment.